lua/nvim-tree.lua

```diff
@@ -666,8 +666,9 @@ function M.setup(conf)
   validate_options(conf)
 
   local opts = merge_options(conf)
-  local netrw_disabled = opts.disable_netrw or opts.hijack_netrw
+  legacy.move_mappings_to_keymap(opts)
```

Comment:
I like this approach: moving the mappings after merge means we only have to do this once.